packages/perseus/src/hints-renderer.tsx

```diff
@@ -49,6 +48,9 @@ class HintsRenderer extends React.Component<Props, State> {
         linterContext: PerseusLinter.linterContextDefault,
     };
 
+    // Store refs to HintRenderer components
```

Comment:
Small suggestion to change to "why" versus "what"
```suggestion
    // Refs needed to call focus() and getSerializedState() on individual hints
```